### PR TITLE
docs: add repository info

### PR DESCRIPTION
**Pull request by aludwiko**
@ennru I haven't added the repository information in all places where we use `@@dependency` directive. Only in cases where it makes sense from my perspective, but maybe I missed sth.

### CODE REVIEW

File: docs/src/main/paradox/cassandra.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ processing semantics, but not exactly-once.
 
 ## Dependencies
 
+The Akka dependencies are available from Akka's library repository. To access them there, you need to configure the URL for this repository.
+
+@@repository [Maven,sbt,Gradle] {
```
Comment (sebastian-alfers) [inline docs/src/main/paradox/cassandra.md:18]: Is the order right here?

Comment (aludwiko) [reply docs/src/main/paradox/cassandra.md]: fixed

**Review by franciscolopezsancho — APPROVED** [review]
LGTM

File: project/plugins.sbt [hunk 2]
```diff
@@ -8,7 +8,7 @@ addSbtPlugin("com.github.sbt" % "sbt-ci-release" % "1.5.12")
 addSbtPlugin("com.lightbend.akka.grpc" % "sbt-akka-grpc" % "2.3.2")
 
 // Documentation
-addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.49")
+addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.51")
```
Comment (franciscolopezsancho) [inline project/plugins.sbt:11]: why this?

**Pull request merged**